scripts/collect-modules.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
M: That should be fine on macOS and Linux, can we run on CI-Windows as well?

Reply:
This is used on macOS only so it's fine.